src/wizard/dotnet/aspnet.md

```diff
@@ -0,0 +1,64 @@
+---
+name: ASP.NET
+doc_link: https://docs.sentry.io/platforms/dotnet/guides/aspnet/
+support_level: production
+type: framework
+---
+
+Install the **NuGet** package:
+
+Package Manager:
```

Comment:
something seems odd here. Do we mean to say "Insatll the **NuGet** package manager:"?

Reply:
like this:

![image](https://user-images.githubusercontent.com/1633368/103961015-90cc2280-5121-11eb-8db9-ecbbf8f36f5f.png)


https://www.nuget.org/packages/Sentry/3.0.0-alpha.9